src/Controls/src/Core/HandlerImpl/VisualElement/VisualElement.cs

```diff
@@ -24,15 +24,11 @@ public partial class VisualElement
 				[nameof(IViewHandler.ContainerView)] = MapContainerView,
 			};
 
-		static CommandMapper<IView, IViewHandler> ControlsViewCommandMapper = new(ViewHandler.ViewCommandMapper)
-		{
-			[nameof(IView.Focus)] = MapFocus,
-		};
-
 		internal static void RemapForControls()
 		{
 			ViewHandler.ViewMapper = ControlsVisualElementMapper;
-			ViewHandler.ViewCommandMapper = ControlsViewCommandMapper;
+
+			ViewHandler.ViewCommandMapper.AppendToMapping<VisualElement, IViewHandler>(nameof(IView.Focus), MapFocus);
```

Comment:
I think we need to change this to a ModifyMapping and then pass the action in to the `ViewExtensions` method. 

Executing the old logic is conditional not always

https://github.com/dotnet/maui/pull/15040/files#diff-0b23daf01dc61f5761c5fbf2bf67bed116f20788abc248c741a4984ef24ffbedL480

Reply:
I think i reversed the order of if statements to reduce nesting... let me re compare...